recipes/skore-local-project/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set package_name = "skore-local-project" %}
+{% set module_name = "skore_local_project" %}
+{% set version = "0.0.2" %}
+
+package:
+  name: {{ package_name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ package_name[0] }}/{{ package_name }}/{{ module_name }}-{{ version }}.tar.gz
+  sha256: 19234b2829f36a8d739991752d1f1df6a45fb3fed72b6476ad5784858d9f7464
+
+build:
+  entry_points:
+    - skore.plugins.project.local = {{ module_name }}:Project
```

Comment:
It's probably better to remove the jinja templating here and in the import test and just write out the module name. Also remove the jinja definition at the top of the file.